pydatastructs/strings/tests/test_trie.py

```diff
@@ -2,13 +2,20 @@
 
 def test_Trie():
 
-    strings = ["A", "to", "tea", "ted", "ten", "i", "in", "inn"]
+    strings = ["A", "to", "tea", "ted", "ten", "i",
+               "in", "inn", "Amfn", "snbr"]
     trie = Trie()
     for string in strings:
         trie.insert(string)
 
+    prefix_strings = ["te", "t", "Am", "snb"]
+
     for string in strings:
+        assert trie.is_inserted(string)
```

Comment:
Let's also test by traversing `strings[::-1]` in a separate loop.

Reply:
Done. Thank you.